libs/designsystem/radio/src/radio.component.ts

```diff
@@ -25,7 +30,7 @@ export class RadioComponent {
 
   @HostBinding('class')
   @Input()
-  size?: 'xs' | 'sm' | 'md';
+  size?: 'xs' | 'sm' | 'md' = 'md';
```

Comment:
Nitty gritty, but I guess this is no longer optional (i.e. can be `undefined`) if it has a default value? 🤔 
```suggestion
  size: 'xs' | 'sm' | 'md' = 'md';
```